controllers/awscluster_controller.go

```diff
@@ -151,6 +169,22 @@ func (r *AWSClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return
 	}
 
+	// We need Spec.IdentityRef to be set, TODO check this
+	if awsCluster.Spec.IdentityRef == nil {
+		return ctrl.Result{}, microerror.Maskf(errors.IdentityNotSetError, "AWSCluster %s/%s does not have Spec.IdentityRef set", awsCluster.Namespace, awsCluster.Name)
+	}
+
+	identity := &capa.AWSClusterRoleIdentity{}
+	identityNamespacedName := types.NamespacedName{
+		Namespace: awsCluster.Namespace,
```

Comment:
awsClusterRoleIdentity is a global resource, there is no need to specify namespace